runtime/src/main/java/org/corfudb/runtime/RebootUtil.java

```diff
@@ -51,13 +61,16 @@ public static void reset(@NonNull String endpoint,
      * @param corfuRuntimeParameters CorfuRuntimeParameters can specify security parameters.
      * @param retries                Number of retries to bootstrap each node before giving up.
      * @param retryDuration          Duration between retries.
+     * @param clusterId              Optional cluster Id. If None is provided, then the default one
+     *                               will be used to create a base client.
      */
     public static void reset(@NonNull String endpoint,
                              @NonNull CorfuRuntimeParameters corfuRuntimeParameters,
                              int retries,
-                             @NonNull Duration retryDuration) {
+                             @NonNull Duration retryDuration,
+                             Optional<UUID> clusterId) {
```

Comment:
The APIs in this file are public APIs consumed by our verticals, you probably need to modify the veritcal codes when you check-in to production code. Is this clusterId field really needed here, can we just use the default?

Reply:
It's a fair point. Unfortunately, a reset message is the one we should check the layout and id for the reset messages. I checked whether this API is used in the newest release, and it seems like it's not (they just use our internal client code instead). For the previous releases this API is used in a few places in the cluster management code.